lib/components/T3Form/T3FormFieldset/T3FormFieldset.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
This component is simple. Should we use the functional component here? Or we completely drop functional components as they won't be supported in vue 3.X?

Reply:
for simplicity we will drop functional components.